src/platforms/android/configuration/integrations/jetpack-compose.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Jetpack Compose
+caseStyle: camelCase
+supportLevel: production
+sdk: sentry.java.compose
+description: "Learn more about the Sentry Compose integration."
+categories:
+  - mobile
+  - desktop
```

Comment:
Should we mention its Android only for now, I see the code under the `androidMain` folder.

Reply:
Yeah, makes sense, I forgot to remove it actually 👍